sdk/cosmos/azure-cosmos/README.md

```diff
@@ -106,10 +104,10 @@ After authenticating your [CosmosClient][ref_cosmosclient], you can work with an
 database_name = 'testDatabase'
 try:
     database = client.create_database(database_name)
-except HTTPFailure as e:
-    if e.status_code != 409:
-        raise
+except errors.CosmosResourceExistsError:
     database = client.get_database_client(database_name)
+except errors.CosmosHttpResponseError:
```

Comment:
This can be removed, no? Catch and immediately unconditionally re-raise fills no purpose, correct?

Reply:
Correct - it was simply there as a demonstration of the base error type. But I'll remove it.